index.html

```diff
@@ -181,64 +188,68 @@ <h1>Multi-Post Stories</h1>
     </section>
 
     <section class="about-myself">
-      <h1 class="header">
-        About Myself
-      </h1>
+      <div class="about-myself-wrapper">
+        <h1 class="header">
+          About Myself
+        </h1>
 
-      <div class="header-text">
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-      </div>
-      
-      <button class="about-myself-link" type="submit">LET'S CONNECT</button>
-
-      <div class="about-myself-icons">
-        <img src="photo/logos/github.jpg" alt="">
-        <img src="photo/navs/Linkedin-icon.jpg" alt="">
-        <img src="photo/logos/angellist.jpg" alt="">
-        <img src="photo/logos/twitter.jpg" alt="">
-        <img src="photo/logos/medium.jpg" alt="">
+        <div class="header-text">
+          <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        
+        <button class="about-myself-link" type="submit">LET'S CONNECT</button>
+
+        <div class="about-myself-icons">
+          <img src="photo/logos/github.jpg" alt="">
+          <img src="photo/navs/Linkedin-icon.jpg" alt="">
+          <img src="photo/logos/angellist.jpg" alt="">
+          <img src="photo/logos/twitter.jpg" alt="">
```

Comment:
- Please work on your about section on the spacing, font-size, color, and alignment of elements on the webpage so it looks closer to the Figma design.

##### Your design 
![Screenshot from 2022-07-16 09-36-28](https://user-images.githubusercontent.com/58771507/179347439-23b97108-6969-41d9-870e-233bd6f30436.png)

##### Figma design
![Screenshot from 2022-07-16 09-37-12](https://user-images.githubusercontent.com/58771507/179347463-9306f7d8-699e-460f-97f2-29d3ca66b5a8.png)